server/src/routes/api/v1/pizzaPlaceReviewRouter.js

```diff
@@ -21,6 +22,26 @@ pizzaPlaceReviewRouter.post("/", async (req, res) => {
     return res.status(500).json({ errors: error })
   }
 })
+  
+pizzaPlaceReviewRouter.put("/:reviewId", async (req, res) => {
+  const body = req.body
+  body.id = req.params.reviewId
+  const formInput = cleanUserInput(body)
+  try {
+    const editedReview = await Review.query().patchAndFetchById(formInput.id, {
+      title: formInput.title,
+      text: formInput.text,
+      rating: formInput.rating
+    })
+    const serializedEditedReview = await ReviewSerialzer.getDetail(editedReview)
+    return res.status(201).json({editedReview: serializedEditedReview})
```

Comment:
status 201 indicates something was created - if we updated an object and are sending that object back we should use status 200 indicating "OK" that the action was successful